### PR TITLE
Add token prices

### PR DESCRIPTION
**Pull request by mvaivre**
For now, we show NaN if a price can't be displayed. This can be improved in a follow up PR.

Closes https://github.com/alephium/alephium-frontend/issues/338

### CODE REVIEW

File: apps/explorer/src/pages/AddressInfoPage/TokenList.tsx [hunk 1]
```diff
@@ -86,6 +91,17 @@ const TokenList = ({ tokens, limit, isLoading, className }: TokenListProps) => {
 
             <TableCellAmount>
               <TokenAmount assetId={token.id} value={token.balance} suffix={token.symbol} decimals={token.decimals} />
+              {token.symbol && token.verified && tokensWithAvailablePrice?.includes(token.symbol) && (
+                <Amount
+                  value={calculateAmountWorth(
+                    token.balance,
+                    tokensPrices.find((p) => p.symbol === token.symbol)?.price || NaN
```
Comment (nop33) [inline apps/explorer/src/pages/AddressInfoPage/TokenList.tsx:98]: nit: Maybe add a TODO here?

**Review by nop33 — COMMENTED** [review]
Code looks good ✅ 
I'll do functional tests once web3 gets released.

**Comment by nop33** [comment]
This PR needs to get rebased/merge on this branch: `fetch-token-prices-rebased`

**Comment by nop33** [comment]
It needs to be rebased again (or master merged into it), sorry :/

**Review by nop33 — COMMENTED** [review]
It needs to be rebased again (or master merged into it), sorry :/ I tried to do it, but I feel like your PR is using endpoints that don't exist.

File: apps/explorer/src/api/assets/assetsHooks.ts [hunk 2]
```diff
@@ -169,3 +169,28 @@ export const useAssetsMetadata = (assetIds: string[] = []) => {
     return returnedVerifiedTokensMetadata
   } else return returnedCompleteMetadata
 }
+
+// TODO: get list from backend (enum?)
+// See: https://github.com/alephium/explorer-backend/issues/512
+export const useTokensWithAvailablePrice = () => ['ALPH', 'USDT', 'USDC', 'DAI', 'WBTC', 'WETH', 'AYIN']
```
Comment (nop33) [inline apps/explorer/src/api/assets/assetsHooks.ts:175]: Why not do the same as in the wallets where we query every verified token? If we get a price, good. If we don't, also good. We keep track of what we queried so we don't query again.

Comment (mvaivre) [reply apps/explorer/src/api/assets/assetsHooks.ts:175]: Just to reduce the number of useless queries

Comment (nop33) [reply apps/explorer/src/api/assets/assetsHooks.ts:175]: ok, explorer is easily update-able in contrary to wallets that require user to upgrade. Approving.

**Review by nop33 — APPROVED** [review]

**Pull request merged**